### ISSUE TITLE
RO terms that normalize but the "invertedness" is not reported

### ISSUE THREAD

**Issue by saramsey** [TranslatorIIPrototypes/EdgeNormalization#7]
When I attempt to normalize `RO:0001022` ("has allergenic trigger"), I get `biolink:causes`, which I think is close, but not quite correct (I think it should be the inverse of that). 

```
curl -X GET "https://edgenormalization-sri.renci.org/resolve_predicate?predicate=RO%3A0001022&version=latest" -H  "accept: application/json"
```

<img width="909" alt="Screen Shot 2020-07-15 at 2 28 33 PM" src="https://user-images.githubusercontent.com/5562107/87601735-8bc67300-c6aa-11ea-958d-cdf04fb59bec.png">

The Relation Ontology (RO) has "inversion" encoded in their hierarchy relationships, as shown in Ontobee:

<img width="284" alt="Screen Shot 2020-07-15 at 2 43 08 PM" src="https://user-images.githubusercontent.com/5562107/87601822-b6b0c700-c6aa-11ea-9632-940c534ea646.png">

were these taken into account in the ETL of RO that was (presumably) done to generate the synonym relationships in the Edge Normalizer?

**Comment by saramsey** [comment]
This is causing inverted relationships in KG2 that lead to strange results, like "shellfish allergy" causes "shellfish".

**Comment by cbizon** [comment]
Ah, this is an excellent point.  We'll get on it.

**Comment by cbizon** [comment]
Well, having dug in a little, and talked to @balhoff I think that this is actually an RO bug.  Both `has allergic trigger` and `has autoimmune trigger` seem to have an incorrect superproperty.  

@saramsey if you know that RO really encodes inverses as part of a property inheritance, can you explain how it's coded there?

I'll file a bug report at RO and see what they say.  Leaving this issue open until I hear from them.

**Comment by cbizon** [comment]
https://github.com/oborel/obo-relations/issues/380

**Comment by saramsey** [comment]
Hi @cbizon thanks for your thoughtful reply. I guess I should restate my claim: I *surmised* that RO encodes this information because it seems to be available in Ontobee (see screencap in the OP), see the plusses and minuses in the treeview. Apologies if I've misinterpreted the meaning of the plusses & minuses.

http://www.ontobee.org/ontology/RO?iri=http://purl.obolibrary.org/obo/RO_0001023

**Comment by saramsey** [comment]
Also thanks to @balhoff for consulting on this issue.

**Comment by cbizon** [comment]
Ok, I think (not 100% sure) that the +/- on ontobee indicates whether the term has children (+) or not (-).

**Comment by balhoff** [comment]
> Ok, I think (not 100% sure) that the +/- on ontobee indicates whether the term has children (+) or not (-).

@cbizon that's right. If there is an inverse property, it is stated in a "Inverse Properties" section further down.
